src/Tests/Microsoft.NET.Rebuild.Tests/Microsoft.NET.Rebuild.Tests.csproj

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@dsplaisted @tannergooding  do we really need the test projects in the SDK repo to be as explicit as it is below?